reference/paths/pci_forward__make-pci-forward.yaml

```
@@ -0,0 +1,59 @@
+---
+summary: Forward PCI data
+operationId: make-pci-forward
+description:
+  Forward an API call to a PCI endpoint. The request body is evaluated and any
+  template fields are replaced by the card data before the request is sent to
+  the given destination.
+tags:
+  - PCI Forward
+
+parameters:
+  - $ref: "../header-parameters/pci_forward_payment_methods.yaml"
+  - $ref: "../header-parameters/pci_forward_url.yaml"
+  - $ref: "../header-parameters/pci_forward_http_method.yaml"
+  - $ref: "../header-parameters/pci_forward_header.yaml"
+  - $ref: "../header-parameters/pci_forward_timeout.yaml"
+
+requestBody:
+  description: Payload to forward in the request.
+  content:
+    "*/*":
+      schema:
+        type: string
+      examples:
+        Make a JSON request:
+          value: '{"number":"{{CARD_NUMBER_1}}","expiry":"{{CARD_EXPIRATION_DATE_1}}"}'
```

Yaml actually supports multi line strings. This would make for a much nicer example.

```suggestion
          value: "{
            "number":"{{CARD_NUMBER_1}}",
            "expiry":"{{CARD_EXPIRATION_DATE_1}}"  
          }"
```

---

Somehow I did not get it to work the first time and `prettier` was messing with it. Fixed now https://github.com/gr4vy/gr4vy-openapi/pull/353/commits/e22b62446062ec59c7466662d820db64ae95ee41